readers-frontend/src/pages/index.astro

```diff
@@ -28,5 +32,14 @@ const featuredArticlesIds: featuredHome | null =
       <BannerEdition />
     </div>
     <Muse />
+    <div class="px-[3%] pt-8 sm:px-[0.5%] 2xl:px-[13%]">
+      {
+        categories.map((category) => (
+          <div class="featured-section">
```

Comment:
Since this class doesn't exist, wouldn't it be better to give each section an ID instead?

Reply:
I'm using it for the tests to check that only one featured section is showing

Reply:
You can do that with an ID

Reply:
do you mean featured-section as an ID or news, comment,..etc for ids?

Reply:
like news-section, comment-section etc

Reply:
Okay let me have a look into it